blog/package.json

```diff
@@ -11,30 +11,31 @@
   "author": "MoleculerJS",
   "license": "MIT",
   "dependencies": {
-    "bluebird": "3.5.1",
-    "chalk": "2.4.1",
-    "eslint": "5.0.1",
-    "express": "4.16.3",
-    "fakerator": "0.3.0",
-    "hashids": "1.1.4",
-    "ioredis": "3.2.2",
-    "lodash": "4.17.10",
-    "moleculer": "0.13.0",
-    "moleculer-console-tracer": "0.1.1",
-    "moleculer-db": "0.7.3",
-    "moleculer-db-adapter-mongoose": "0.7.0",
-    "moleculer-fake": "1.0.11",
-    "moleculer-repl": "0.5.0",
-    "moleculer-web": "0.8.0",
-    "moment": "2.22.2",
-    "mongoose": "5.2.1",
-    "morgan": "1.9.0",
-    "nats": "1.0.0",
-    "pug": "2.0.3",
-    "slugify": "1.3.0"
+    "bluebird": "^3.7.2",
```

Comment:
Bluebird is used in the project?

Reply:
No, I've removed it in https://github.com/moleculerjs/moleculer-examples/pull/22/commits/d226aa09a9302e9dd27e55c4f6291aa5fc555a22